### PR TITLE
Add conda to docu

### PR DESCRIPTION
**Pull request by mschwoer**
- add commands on how to create conda env to docs,
- revert not using pip cache in gh actions
- add pre-commit to docs
- add release template

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -13,3 +20,18 @@ You need to specify the commit to release, and the release tag (e.g. `vX.Y.Z`).
...
 6. Run the [Publish Docker Image](https://github.com/MannLabs/alphadia/actions/workflows/publish_docker_image.yml) workflow,
 specifying the release tag (e.g. `vX.Y.Z`).
+
+
+## Notes for developers
+### pre-commit hooks
+It is highly recommended to use the provided pre-commit hooks, as the CI pipeline enforces all checks therein to
+pass in order to merge a branch.
+
+The hooks need to be installed once by
```
Comment (GeorgWa) [inline docs/contributing.md:30]: Should we add that they are a dependency in the `[dev]` install?

File: docs/installation.md [hunk 2]
```diff
@@ -30,10 +30,20 @@ you can install alphaDIA via `pip`.
...
+If you want to use `.raw` files on Thermo instruments alphaRaw is required, which depends on Mono.
+A detailed guide to installing alphaRaw with mono can be found [here](https://github.com/MannLabs/alpharaw#installation), a short version is given
+[below](#2-installing-mono).
 
 ### 2. Setting up the environment
 
+It is highly recommended to use a conda environment for the installation of alphaDIA.
+This ensures that all dependencies are installed correctly and do not interfere with other packages.
+```bash
+conda create -n alphadia python=3.11
```
Comment (GeorgWa) [inline docs/installation.md:42]: Let's change the default to `alpha`. This is the default environment name expected by the dev install of the gui.

**Review by GeorgWa — APPROVED** [review]
LGTM

File: docs/installation.md (hunk 2 above)
Comment (mschwoer) [reply docs/installation.md:42]: could we set "envName": "alphadia" instead in the gui install?

Comment (GeorgWa) [reply docs/installation.md:42]: Absolutely 👍

**Pull request merged**